app/views/clinical_resource_hubs/show.html.erb

```diff
@@ -1,8 +1,34 @@
-<div>
-  <%# debugger %>
-  VISN NUmber: <%= params["id"] %>
-  <div>
-  VISN_ID: <%= @crh.visn_id %>
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :body_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% visn_number = params["number"]  %>
+<% visn_id = @crh.visn_id %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag '_practice_card_utilities', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+
+    <%= render partial: 'crh_show', formats: [:js] %>
+    <%# set visnNumber before loading the crh/show file %>
+    var visnNumber = <%= visn_number %>;
+  <% end %>
+  <%= javascript_include_tag 'clinical_resource_hubs/crh_show', 'data-turbolinks-track': 'reload' %>
+<% end %>
+
+<div id="crh-show" class="grid-col-12" data-reload="true">
```

Comment:
the style on this page is slightly off from the Figma design

Figma
<img width="940" alt="Screen Shot 2022-03-21 at 12 08 01 PM" src="https://user-images.githubusercontent.com/20211771/159333647-0fc03a38-63f2-44b1-ae0d-677cb929609d.png">

This branch
<img width="1916" alt="Screen Shot 2022-03-21 at 12 50 07 PM" src="https://user-images.githubusercontent.com/20211771/159333797-32a181b7-a472-48be-b0d6-d74eb35cb9b2.png">
   
Things I am noticing:

- no definition underline for "Clinical Resource Hub"
- not enough padding for breadcrumbs
- different font for text below heading
-

Reply:
Thx for your help on this one Elisa.